### DISCUSSION TITLE
Python program to wrap the CLI program

### DISCUSSION THREAD

**Discussion by KJHJason** [KJHJason/Cultured-Downloader#20] [Ideas]
I might convert this repo to wrap around the [CLI version of Cultured Downloader](https://github.com/KJHJason/Cultured-Downloader-CLI) for much better usability and user experience. 

Since the CLI version has much better download capabilities such as better ugoira ffmpeg conversion, it should be a wise choice instead of revamping the whole program again.

However, until then, I am currently busy with school assignments so you will only see this come to light in 2-3+ months' time.

**Comment by avan-pra** [comment]
Hold on, am i the only one who got ghost pinged by a bot/scam account ? (sorry it's not related to the issue at all)

**Reply by KJHJason** [reply]
> Hold on, am i the only one who got ghost pinged by a bot/scam account ? (sorry it's not related to the issue at all)

nah you're not the only one, I got an email as well

**Comment by KJHJason** [comment]
Instead of wrapping the CLI program, I've decided to work on a GUI version using Golang as well.
However, this will take quite a while to develop as I'm currently porting all the logics of [Cultured Downloader CLI](https://github.com/KJHJason/Cultured-Downloader-CLI) into another github repository so that the GUI and the CLI programs will share the same logic for easier maintainability.

Additionally, school is about to start so this will probably take a few months.
